js/chips.js

```diff
@@ -92,38 +129,49 @@ function userClick(element) {
 	//takes a parameter of e.target, any clicked element on the page
 
 	// if the clicked element is also the selectedChip, do nothing.
-	if (element == selectedChip) {
-		// this condition will execute first and the return false here will ensure that the remainder 
+	// or if the clicked element is the selectedSquare, do nothing. This might change, if we want to display a message to the user.
```

Comment:
Right now the function will not execute if user tries to select the same square again. This is same as for chips. However, we may want to change this, if we want to _do something_, for example show the same alert/error message to the user again. What do you think? Which option is more clear and better for the user?